src/components/KInput/KInput.vue

```diff
@@ -301,14 +300,14 @@ $kInputSlotSpacing: var(--kui-space-40, $kui-space-40); // $kSelectInputSlotSpac
 
 /* Component styles */
 
-.k-input-wrapper {
+.k-input {
   display: flex;
   flex-direction: column;
   width: 100%;
 
   // error styles
   &.input-error {
-    .k-input, .k-input[type="file"] {
+    .input, .input[type="file"] {
```

Comment:
I see it’s already merged, but really the class isn’t needed; you could just use the `.k-input input` native selector, and then specific styles if needed based on the `type`

Reply:
I know but I decided still having class there won't hurt